src/vm/threadsuspend.cpp

```diff
@@ -8342,7 +8342,7 @@ void ThreadSuspend::SuspendEE(SUSPEND_REASON reason)
 #endif //TIME_SUSPEND
 }
 
-#if defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
+#if !defined(_TARGET_ARM64) && defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
This should not ifdefed out for TARGET_ARM64. If there are problems in this code that you do not want to fix right now, disable FEATURE_HIJACK in the central CMakeFile and add TODO next to it.